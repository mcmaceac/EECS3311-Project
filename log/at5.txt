add_user(1, "DDos")
add_user(2, "Dbos")
list_users
add_group(10000000000,"long")
