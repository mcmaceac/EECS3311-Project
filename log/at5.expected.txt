  0:  OK

->add_user(1,"DDos")
  1:  OK
  Users:
      1->DDos
  Groups:
  Registrations:
  All messages:
  Message state:

->add_user(2,"Dbos")
  2:  OK
  Users:
      1->DDos
      2->Dbos
  Groups:
  Registrations:
  All messages:
  Message state:

->list_users
  3:  OK
  1->DDos
  2->Dbos

->add_group(10000000000,"long")
  4:  OK
  Users:
      1->DDos
      2->Dbos
  Groups:
      1410065408->long
  Registrations:
  All messages:
  Message state:

